content/en/docs/Reference/catalog-templates.md

```diff
@@ -503,3 +503,146 @@ schema: olm.channel
+- Where each input (`Component`) exists
+- How each contribution is mapped to a destination catalog (`Component.Name`). All destination catalogs defined in this file **must** also exist in the catalog configuration file created by the catalog maintainers
+- How each `Component` is processed to generate a catalog contribution (`Component.Strategy`)
+- The directory structure the catalog contribution generation should follow (`Component.Destination`)
+
+The cue schema for the `olm.composite` schema is:
+```cue
+#CompositeConfig: {
+	Schema: "olm.composite"
+	Components: [...#Component]
+}
+
+#Component: {
+	Name:        string
+	Destination: #ComponentDestination
+	Strategy:  #BuildStrategy
+}
+
+#ComponentDestination {
+	Path: string
+}
+
+#BuildStrategy: {
+	Name:   string
+	Template: #TemplateDefinition
+}
+
+#TemplateDefinition: {
+	Schema: string
+}
+```
+
+#### Catalog Maintainer
+The `olm.composite.catalogs` schema represents the catalog maintainer role and defines the following:
+- One or more catalogs (`Catalog`) identified by a string (`Catalog.Name`)
+- Supported input formats (`Builder`) for each `Catalog` (`Catalog.Builders`) which is a list of strings where each item is a `Builder` schema. The currently supported `Builder` schemas are:
+    - `olm.builder.basic` which represents that the use of [basic templates](#basic-template) for catalog contribution generation is allowed
+    - `olm.builder.semver` which represents that the use of [semver templates](#semver-template) for catalog contribution generation is allowed
+    - `olm.builder.raw` which represents that the use of raw FBC for catalog contribution generation is allowed
+    - `olm.builder.custom` which represents that the use of custom catalog contribution generation logic is allowed
+- The expected directory structure of any generated contribution (`Catalog.Destination.WorkingDir`)
+
+The cue schema for the `olm.composite.catalogs` schema is:
+```cue
+#CatalogConfig: {
+	Schema: "olm.composite.catalogs"
+	Catalogs: [... #Catalog]
+}
+
+#Catalog: {
+	Name:        string
+	Destination: #CatalogDestination
+	Builders: [...string]
+}
+
+#CatalogDestination: {
+	BaseImage:  string
+	WorkingDir: string
+}
+```
+
+### Examples
+**catalogs.yaml**
+
+The following example specifies a catalog configuration that defines a catalog named `v1`. This example limits the input to the [basic template](#basic-template) for contribution generation logic. If you use the `opm alpha render-template composite` subcommand the path to this file can be specified with the `-f` option. This option can be a file path or a URL. If you use a URL, the URL must return the raw file contents.
+
+```yaml
+schema: olm.composite.catalogs
+catalogs:
+- name: v1 {{< code_callout 1 >}}
+  destination:
+    baseImage: quay.io/operator-framework/opm:v1.24
+    workingDir: catalogs/v1 {{< code_callout 2 >}}
+  builders:
+    - olm.builder.basic {{< code_callout 3 >}}
```

Comment:
I know it's a huge ask, but since elsewhere we're really trying to evangelize the semver template, could you remodel this to use semver instead of basic?

Reply:
https://github.com/operator-framework/olm-docs/pull/286/commits/8c89bd367d6835faed14561c8660ad7acb18e3fc